### PR TITLE
Alex's branch

### PR DESCRIPTION
**Pull request by apapiez**
*Merge Me Senpai*

### CODE REVIEW

File: Assets/Editor/PieChartControllerEditor.cs [hunk 1]
```diff
@@ -0,0 +1,23 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEditor;
+
+
+[CustomEditor(typeof(PieChartController))]
```
Comment (Thalpy) [inline Assets/Editor/PieChartControllerEditor.cs]: Comments pls

File: Assets/Scripts/Editor/GameMasterEditor.cs [hunk 2]
```diff
@@ -0,0 +1,21 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEditor;
+
+
+[CustomEditor(typeof(GameMaster))]
```
Comment (Thalpy) [inline Assets/Scripts/Editor/GameMasterEditor.cs:7]: Still no comments classic. Add a summary please

File: Assets/Scripts/GameMaster.cs [hunk 3]
```diff
@@ -13,8 +13,11 @@ public class GameMaster : MonoBehaviour
     public static CrisisMaster crisisMaster;
     public static JL_HandController handController;
     public static CardMaster cardMaster;
+
     public List<Effect> effects = new List<Effect>();
 
+    public PieChart pieChart;
+
     
     // Start is called before the first frame update
```
Comment (Thalpy) [inline Assets/Scripts/GameMaster.cs:22]: A classic comment

File: Assets/Scripts/PieChart.cs [hunk 4]
```diff
@@ -0,0 +1,85 @@
...
+        {
+            totalValue += valuesToSet[i];
+        }
+
+        return valuesToSet[index] / totalValue;
+        
+    }
+
+
+    // Start is called before the first frame update
```
Comment (Thalpy) [inline Assets/Scripts/PieChart.cs:74]: Consider stripping these in the final review

**Review by Thalpy — APPROVED** [review]

File: Assets/Scripts/GameMaster.cs (hunk 3 above)
Comment (apapiez) [reply Assets/Scripts/GameMaster.cs:22]: yes indeed

**Pull request merged**